### PR TITLE
[e2e_test]: support simple e2e_test

### PR DESCRIPTION
**Pull request by hanfei1991**
after starting docker compose by 
```
docker-compose -f 3m3e.yaml -f demo.yaml up
```
 we can run this simple e2e_test by running `go test` in the e2e directory

### CODE REVIEW

File: cmd/demoserver/kvdatamake.go [hunk 1]
```diff
@@ -281,6 +284,7 @@ func (s *DataRWServer) ReadLines(req *pb.ReadLinesRequest, stream pb.DataRWServi
 		case <-s.ctx.Done():
 			return nil
 		default:
+			time.Sleep(time.Millisecond * 5)
```
Comment (nongfushanquan) [inline cmd/demoserver/kvdatamake.go:287]: why sleep here

File: jobmaster/cvsJob/cvsJobMaster.go [hunk 2]
```diff
@@ -114,26 +115,28 @@ func (jm *JobMaster) Tick(ctx context.Context) error {
...
-			log.L().Info("sync file finished ", zap.Any("message", worker.file))
 		} else if status.Code == lib.WorkerStatusError {
 			log.L().Error("sync file failed ", zap.Any("message", worker.file))
 		} else {
 			log.L().Info("worker status abnormal", zap.Any("status", status))
 		}
 	}
 	log.L().Info("cvs job master status  ", zap.Any("id :", jm.workerID), zap.Int64("counter: ", jm.counter))
-	return nil
+	err := jm.BaseJobMaster.UpdateJobStatus(ctx, jm.Status())
```
Comment (nongfushanquan) [inline jobmaster/cvsJob/cvsJobMaster.go:134]: don't call updatejobstatus frequently, for the updateJobstatus will store the status into the megastore.   Only call it when the status changes

File: cmd/demoserver/kvdatamake.go (hunk 1 above)
Comment (hanfei1991) [reply cmd/demoserver/kvdatamake.go:287]: Slower the read speed to mitigate the pressure of disk.

File: jobmaster/cvsJob/cvsJobMaster.go (hunk 2 above)
Comment (hanfei1991) [reply jobmaster/cvsJob/cvsJobMaster.go:138]: fixed, PTAL

Comment (hanfei1991) [reply jobmaster/cvsJob/cvsJobMaster.go]: updated, PTAL again

**Review by nongfushanquan — APPROVED** [review]

**Pull request merged**